ignite/cmd/generate_vuex.go

```diff
@@ -10,7 +10,7 @@ import (
 func NewGenerateVuex() *cobra.Command {
 	c := &cobra.Command{
 		Use:     "vuex",
-		Short:   "Generate Typescript client and Vuex stores for your chain's frontend from your `config.yml` file",
+		Short:   "*DEPRECATED* Generate Typescript client and Vuex stores for your chain's frontend from your `config.yml` file",
```

Comment:
If you're agree on removing the command from the help : 

```suggestion
		Short:   "Generate Typescript client and Vuex stores for your chain's frontend from your `config.yml` file",
		Deprecated: "Use `ignite generate composable` instead.",
```

An other possibility is too remove that file and add the `vuex` command to the `deprecated()` func located in `ignite/cmd/cmd.go`.

Reply:
What is the preferred way of doing it?

Essentially the command should remain there and useable as it's used by people and it's more of a notification that it will be deprecated at some point in the future

Reply:
OK in that case we shouldn't set the `Deprecated` field. I resolve this comment.